elasticdl/python/tests/parameters_test.py

```diff
@@ -106,3 +106,23 @@ def test_set_embedding_param(self):
 
         with self.assertRaises(ValueError):
             self.params.set_embedding_param("tom", [0, 1, 2], values)
+
+    def test_check_grad(self):
+        self._clear_params()
+        self.params.init_from_model_pb(self.model_pb)
+
+        grad0 = Tensor(name="z")
+        with self.assertRaises(ValueError):
```

Comment:
For all the `assertRaises` checks, it would be better to check a regex of the error message so we know which exception we are testing against

Reply:
Done